packages/viewer/src/scenes/colorMaterial.ts

```diff
@@ -0,0 +1,34 @@
+import { Color } from '@vertexvis/utils';
+
+/**
+ * The `ColorMaterial` interface is here.
+ */
+export interface ColorMaterial {
+  opacity: number;
+  glossiness: number;
+  ambient: Color.Color;
+  diffuse: Color.Color;
+  specular: Color.Color;
+  emissive: Color.Color;
+}
+
+export const fromHex = (hex: string, opacity?: number): ColorMaterial => {
+  const color: Color.Color = Color.fromHexString(hex);
+
+  return {
+    opacity: 100,
+    glossiness: opacity || 10,
+    ambient: {
+      ...color,
+    },
+    diffuse: {
+      ...color,
+    },
```

Comment:
AFAIU, colors should only affect `diffuse`. The others are fixed values.

Reply:
If I didnt pass all of the values, it did not work.

Reply:
Ya, but I don't think we should set `ambient`, `specular`, `emissive` to `color`. Those should probably be some constants that are hardcoded.